GeeksCoreLibrary/Modules/Templates/Controllers/TemplatesController.cs

```diff
@@ -15,6 +15,7 @@
 using GeeksCoreLibrary.Core.Interfaces;
 using GeeksCoreLibrary.Modules.Databases.Interfaces;
```

Comment:
Volgens mij is deze `using` nu niet meer nodig nu je `IDatabaseConnection` eruit hebt gehaald. Kun je de using ook nog weghalen?

Reply:
Goed gezien, heb ik nu weggehaald.